src/routes/moderation.rs

```diff
@@ -54,9 +55,18 @@ pub async fn mods(
             issues_url: m.issues_url,
             source_url: m.source_url,
             status: ModStatus::Processing,
+            license: License {
+                id: "".to_string(),
+                name: "".to_string(),
+                url: None,
+            },
```

Comment:
This needs to have filled out data for moderation to be usable. This should use the same system as the normal mods route if possible - it will be slower, but you should be able to query a list of ids and then use the exact same code as used in the mods route, so you end up with the full information here.

Reply:
This isn't needed, as this is just a list of mods that the moderator needs to approve. When reviewing a mod, a mod will then click on the mods on the frontend and go to the mod's page, which will have this data.

Reply:
How the *frontend* is structured shouldn't affect the backend design.  If this isn't going to be included, make a new structure for it that doesn't include the data that isn't included.  This route should either return the *exact* same information that the normal mods route does, or return *only* a list of mod ids that need to be reviewed, which would then require sending it to the mods route to get the actual information.

This either has to return the full data, or make a new partial mod data structure that specifies what information is actually being sent instead of filling it in with defaults.